Added snippet for google auth (#2057)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/browser-features.md

*  `edge://supervised-user-internals`
*  `edge://version`

<!-- ====================================================================== -->
## Google Authentication

> Google Authentication is disabled in WebView2 due to a policy set by Google. You can read more about this decision from [Google's Blog](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html), and stay up-to-date on the latest in our [WebView2 GitHub Issue](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1647).

Reviewer: ```suggestion
Google Authentication is disabled in WebView2.  See Google's blog: [Upcoming security changes to Google's OAuth 2.0 authorization endpoint in embedded webviews](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html).

To stay up-to-date on the latest discussion, in the WebView2Feedback repo, see [Google Auth Flows and WebView2](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1647).
```

*  `edge://supervised-user-internals`
*  `edge://version`

<!-- ====================================================================== -->
## Google Authentication

> Google Authentication is disabled in embedded webviews, which includes WebView2, due to a security policy set by Google.  See [Upcoming security changes to Google's OAuth 2.0 authorization endpoint in embedded webviews](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html).

Reviewer 2: Can we move the primary actor in this case up to the front of the sentence? Just to make it extra clear to folks that complaining to us won't help?

```suggestion
Google has disabled Google Authentication in embedded webviews, which includes WebView2, due to a security policy they have set.  See [Upcoming security changes to Google's OAuth 2.0 authorization endpoint in embedded webviews](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html).
```
